src/data/ghost.ts

```diff
@@ -3,7 +3,7 @@ import GhostContentAPI from "@tryghost/content-api";
 // Create API instance with site credentials
 const api = new GhostContentAPI({
   url: 'https://overthought.ghost.io',
-  key: 'a731438732f44c2cdf30dd4955',
+  key: process.env.GHOST_API_KEY,
```

Comment:
add Ghostapikey to now 'env' and now 'build env' config  for product. and to  .env and .env.build  for dev. But alway get error missing ghost content api key . Did your work?

Reply:
Oh, also add secret.